controlpanel/settings/common.py

```diff
@@ -555,3 +555,11 @@
 AIRFLOW_SECRET_KEY = os.environ.get('AIRFLOW_SECRET_KEY')
 AIRFLOW_FERNET_KEY = os.environ.get('AIRFLOW_FERNET_KEY')
 
+
+# -- Google Analytics
+if ENV == 'alpha':
+    GOOGLE_ANALYTICS_ID = os.environ.get('GOOGLE_ANALYTICS_ID', 'UA-151666116-2')
+elif ENV == 'prod':
+    GOOGLE_ANALYTICS_ID = os.environ.get('GOOGLE_ANALYTICS_ID', 'UA-151666116-3')
+else:
+    GOOGLE_ANALYTICS_ID = os.environ.get('GOOGLE_ANALYTICS_ID', 'UA-151666116-4')
```

Comment:
I'm assuming these IDs are not sensitive - but generally the correct place to put things like API keys is in the https://github.com/ministryofjustice/analytics-platform-config encrypted repo, and then inject those values into the deployed environment in https://github.com/ministryofjustice/analytics-platform-helm-charts

Reply:
I think this will work for different envs.